frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -37,6 +37,7 @@
         vm.sortColumn = 'rank';
         vm.reverseSort = false;
         vm.columnIndexSort = 0;
+        vm.submissionsExhausted = true;
```

Comment:
Please initialize with `false` value.

Reply:
@RishabhJain2018 
This is for when submissions are remaining **but** `Upload File` is not selected
![screenshot from 2019-01-16 02-17-34](https://user-images.githubusercontent.com/3920286/51208944-076c2600-1935-11e9-837d-772449f69392.png)